tests/integration-tests/configs/common/common.yaml

```diff
@@ -827,9 +747,9 @@ log_rotation:
     dimensions:
       - regions: ["ap-south-1"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_COMMERCIAL_X86 }}
+        oss: ["ubuntu2004"]
         schedulers: ["slurm"]
       - regions: ["ap-northeast-1"]
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
```

Comment:
The feature seems to be architecture independent

Reply:
cool, I'll optimize them.

Reply:
Done in https://github.com/aws/aws-parallelcluster/pull/5201